docker/run_jakartaeetck.sh

```diff
@@ -15,6 +15,16 @@
 # https://www.gnu.org/software/classpath/license.html.
 #
 # SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+function pre_start_domain {
+  GLASS_FISH_CUR_HOME=$1
+  mkdir -p ${CTS_HOME}/monitoring-core/
+  mv ${GLASS_FISH_CUR_HOME}/glassfish/modules/monitoring-core.jar ${CTS_HOME}/monitoring-core/
```

Comment:
Nice to see a pull request including this change, however I think the underlying bug was fixed by https://github.com/eclipse-ee4j/glassfish/pull/23264